### PR TITLE
feat(roll): roll to the ToT 1617207797000

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: playwright/_impl/_frame.py [hunk 1]
```diff
@@ -181,6 +181,21 @@ async def continuation() -> Optional[Response]:
...
+        self,
+        url: URLMatch,
+        wait_until: DocumentLoadState = None,
+        timeout: float = None,
+    ) -> None:
+        matcher = URLMatcher(url)
+        if matcher.matches(self.url):
+            await self.wait_for_load_state(state=wait_until, timeout=timeout)
+            return
+        async with self.expect_navigation(
```
Comment (yury-s) [inline playwright/_impl/_frame.py:194]: very odd alignment :/

**Review by yury-s — APPROVED** [review]
Where are all the tests for the new API that you mentioned?

File: playwright/_impl/_frame.py (hunk 1 above)
Comment (pavelfeldman) [reply playwright/_impl/_frame.py:194]: it isn't me though.

Comment (mxschmitt) [reply playwright/_impl/_frame.py:194]: joke of the day: _seems like [black](https://github.com/psf/black) magic_

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**